lib/state/action/pokemon_actions.dart

```diff
@@ -1,29 +1,19 @@
 import 'dart:async';
-import 'package:async_redux/async_redux.dart';
 import 'package:pokedex_async_redux/api/api_service.dart';
+import 'package:pokedex_async_redux/state/action/actions.dart';
 import 'package:pokedex_async_redux/state/app_state.dart';
 import 'package:pokedex_async_redux/utilities/constants.dart';
 
 /// Getting of pokemons from pokemon api
-class GetPokemonsAction extends ReduxAction<AppState> {
-  @override
-  Future<AppState> reduce() async {
-    final pokemons = await ApiService().pokemonApi.getPokemonList(offset: offset, limit: limit);
-
-    return state.copyWith(pokemons: pokemons);
-  }
-}
-
-/// Getting the pokemon Details from pokemon api
-class GetPokemonDetailsAction extends ReduxAction<AppState> {

```

Comment:
why removed this one? you only need to update `GetPokemonsAction` right? 😱

Reply:
i was just testing it po xd , i'll just restore it

Reply:
resolved